homework9/hw9task2/tests/test_context_manager.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from iterators_2.context_manager import SuppressorClass, suppressor_generator
+
+
+def test_suppressor_as_generator_index_error_positive():
+    with suppressor_generator(IndexError):
+        ["a", "b", "c"][4]
```

Comment:
Why not just raise an error?

Reply:
Because IndexError is suppressed by suppressor_generator (
 `except exc_type:`
           `pass`)

Reply:
`["a", "b", "c"][4]` ->  `raise IndexError()`